proxies/events-federation-gateway/router.yaml

```diff
@@ -0,0 +1,30 @@
+#
+# supergraph: Configuration of the Supergraph server
+#
+supergraph:
+  # The socket address and port to listen on
+  # listen: "${env.MY_LISTEN_ADDRESS}"
+  listen: 0.0.0.0:4000
+  introspection: true
+homepage:
+  enabled: false
+sandbox:
+  enabled: true
+cors:
+  # Set to true to allow any origin
+  # (Defaults to false)
+  allow_any_origin: true
+
+  # List of accepted origins
+  # (Ignored if allow_any_origin is true)
+  # (Defaults to the Apollo Studio url: `https://studio.apollographql.com`)
+  #
+  # An origin is a combination of scheme, hostname and port.
+  # It does not have any path section, so no trailing slash.
+  # origins:
+  # - http://localhost:3000
+  # - https://studio.apollographql.com # Keep this so Apollo Studio can run queries against your router
+  # match_origins:
+  # - "https://([a-z0-9]+[.])*api[.]example[.]com" # any host that uses https and ends with .api.example.com
+override_subgraph_url:
+  cms: ${env.GATEWAY_CMS_ROUTING_URL}
```

Comment:
Is this build time or run time variable?

Reply:
If this is build time variable is any change convert it to run time variable?

Reply:
Lets see... Actually I just understood that I need to override all of the endpoints URL's because we have prod and staging envs. This is probably not the right way to do this after all. I need to study this a bit more - there is probably some directive that a client could give to route the query to desired location, or something like that.

Reply:
oh, and it's runtime variable as far as I know, because the docker image contains a compiled Rust-executable and it's given as environment variable to the docker composer. Nothing is built when the variable is changed